terraform/cronjob.tf

```
@@ -0,0 +1,21 @@
+resource "aws_iam_policy" "cronjob" {
+  name = "cronjob"
```

idealmente hay que restringir esto a un sólo cronjob para no darle privilegios innecesarios a los demás.

```suggestion
resource "aws_iam_policy" "plagiarism_detector_cronjob" {
  name = "plagiarism-detector-cronjob"
```

también hay necesidad de crear el `aws_iam_policy_document`, el `aws_iam_role` y el `kubernetes_service_account` así como https://github.com/omegaup/prod/blob/main/terraform/db-migrate.tf

---

Listo. Apliqué los cambios. Porfa, si encuentras alguna referencia de cómo bajarme ahora los envíos desde S3, me la pasas.